### ISSUE TITLE
/bin/sh: /src/docker-letsencrypt-nginx-proxy-companion-master/install_simp_le.sh: not found

### ISSUE THREAD

**Issue by BMWfan** [buchdag/multiarch-letsencrypt-nginx-proxy#7]
Hello,

i'am using the 2 container method but get the error as described in the subject field.

Greetings

Daniel

**Comment by Snoobz** [comment]
Hello,

I am facing the same exact problem, did you find a solution to this?

Thank you!

**Comment by BMWfan** [comment]
> Hello,
> 
> I am facing the same exact problem, did you find a solution to this?
> 
> Thank you!

Yes, it runs. It tooks me the whole day. 
Now, lets come to the solution so that you can use it as well.

First, clone the actual multiarch-letsencrypt-nginx-proxy repo by using this command:
`git clone https://github.com/buchdag/multiarch-letsencrypt-nginx-proxy.git`

Change to multiarch-letsencrypt-nginx-proxy/nginx-proxy-2containers and delete the following folders:
```
letsencrypt-nginx-proxy-companion
nginx-proxy
```

Clone the newest letsencrypt-nginx-proxy-companion repo by using this command:
`git clone https://github.com/nginx-proxy/docker-letsencrypt-nginx-proxy-companion.git`

Download, extract and rename the updated nginx-proxy version from @tkw1536 which he developed for our scenario with this command:
```
wget https://github.com/KWARC/nginx-proxy/archive/build-from-scratch.tar.gz
tar xzvf build-from-scratch.tar.gz
mv nginx-proxy-build-from-scratch nginx-proxy
```

Now, is everything prepared, you can create your containers by using the command ` docker-compose up -d`

Please let me know if everything works as described.

Greetings

Daniel

**Comment by Snoobz** [comment]
Hello,

Thank you, I'm still facing some issue, specially regarding the let's encrypt container.
I get those errors :

Warning: '/etc/acme.sh' does not appear to be a mounted volume.
Error: can't access to '/etc/acme.sh' directory !
Check that '/etc/acme.sh' directory is declared as a writable volume.

Did you face those too? 
Thanks.

**Comment by buchdag** [comment]
Hi @Snoobz and @BMWfan.

I need to update the repo for the new v2 version of `nginx-proxy/docker-letsencrypt-nginx-proxy-companion`.

I'll try to do it this week.

**Comment by phantomski77** [comment]
Hi @Snoobz and @BMWfan,

have a look at this thread:

https://github.com/nginx-proxy/docker-letsencrypt-nginx-proxy-companion/issues/714#issue-744929237

@buchdag very kindly provided me with the instructions some time ago and I have pushed the resulting builds to Docker Hub (see final post there).

Can't wait for proper v2 (thanks for that!), but in the meantime, you can pull these ready-made. Hope this helps.

Cheers.

**Comment by Snoobz** [comment]
Hello @phantomski77,

I doesn't seem like I'm facing the same issue has you were. I guess I'll wait for the new update from @buchdag.
Thank you very much in advance for the work you will put into this project update!

Have a good day!

**Comment by BMWfan** [comment]
Hello together,

any news about that?

@buchdag if I can help something by testing or so, please let me know.

Another question what I have is: I want to use the certificates for internal dashboards but with public domain. So everytime if the certificate will expire soon I manually port forward (on my router) my external ip (to which my domain points), to my internal docker host and then I get the certificate. I know a possibility like an API to call my router to activate and deactivate the port forwarding. Can you tell me at which position in your solution can I place this call, that it will run for every try to get a certificate? 

Greetings Daniel

**Comment by buchdag** [comment]
@BMWfan sorry I had way more work than anticipated last month, I'll try to get to that ASAP. The companion itself is natively available for multiple archs from DockerHub so there is no need to build it locally anymore. I intend to do the same for `ningx-proxy` and `docker-gen` as soon as we've cleared https://github.com/nginx-proxy/nginx-proxy/issues/1304, so this repo (`buchdag/multiarch-letsencrypt-nginx-proxy`) might not even be needed soonish.

> I know a possibility like an API to call my router to activate and deactivate the port forwarding. Can you tell me at which position in your solution can I place this call, that it will run for every try to get a certificate?

If I had to do something similar, I'd modify the companion bash service loop to make the API call when a renewal is due, wait for the response / confirmation, trigger the certificate renewal, then do the second API call after the ACME client execution ends.
